demo/site/server.ts

```diff
@@ -37,9 +36,9 @@ app.prepare()
                 res.statusCode = 500;
                 res.end("internal server error");
             }
-        }).listen(port, (err) => {
-            if (err) throw err;
```

Comment:
This error handling I removed due to typing problems and [here](https://github.com/vercel/next.js/blob/canary/examples/custom-server/server.ts#L16) they also don't have it